src/feature_lib.cpp

```diff
@@ -0,0 +1,28 @@
+#include "feature_lib.h"
+#include "conversions.h"
```

Comment:
You need to include `opencv2/imgproc.hpp` -- currently you are getting `cv::COLOR_RGB2GRAY` through a transitive dependency from `conversions.h` which is bad

Reply:
done